doc/rst/usingchapel/chplenv.rst

```diff
@@ -489,17 +490,17 @@ CHPL_GMP
 CHPL_HWLOC
 ~~~~~~~~~~
    Optionally, the ``CHPL_HWLOC`` environment variable can select between
-   no hwloc support or using the hwloc package distributed with Chapel in
-   third-party.  Note that hwloc is only used by the qthreads tasking layer,
-   and does not need to be built for other tasking layers.  Current options
-   are:
+   no hwloc support, using the hwloc package distributed with Chapel in
+   third-party, or using a system jemalloc.
 
-       ======= ==============================================================
-       Value   Description
-       ======= ==============================================================
-       none    do not build hwloc support into the Chapel runtime
-       hwloc   use the hwloc distribution bundled with Chapel in third-party
-       ======= ==============================================================
+       ======== ==============================================================
+       Value    Description
+       ======== ==============================================================
+       none     do not build hwloc support into the Chapel runtime
+       system   use a system install of hwloc
```

Comment:
I don't have a problem moving it to a technote, other than that I'm getting tired of working on this PR (why is it so hard to do something so simple?). Also, I'd feel I'd have to link the technote from chplenv.rst and then it might as well just be in chplenv.rst. I'm not sure I see any real benefit to the material being in a technote - other than that it's harder to find. If we want the docs to include a warning, let's put a warning.

What about simply putting in a warning in chplenv.rst that says something like "Use a system install of hwloc at your own risk. Chapel is only tested with the bundled one." (or some other "Use at your own risk. Your mileage may vary." warning). ?